<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="gFileInfo">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>File name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>File type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Record type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Frequency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time start</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time stop</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bDelRow">
      <property name="text">
       <string>‚ùå</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="aSpectrograms"/>
    <addaction name="aCorrelations"/>
   </widget>
   <widget class="QMenu" name="menuArchive">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="aLoadFiles"/>
    <addaction name="aFileStitching"/>
   </widget>
   <widget class="QMenu" name="menuViewer">
    <property name="title">
     <string>Viewer</string>
    </property>
    <addaction name="aSpectrumViewer"/>
   </widget>
   <addaction name="menuArchive"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuViewer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aSpectrograms">
   <property name="text">
    <string>Spectrograms</string>
   </property>
  </action>
  <action name="aCorrelations">
   <property name="text">
    <string>Correlations</string>
   </property>
  </action>
  <action name="aLoadFiles">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="aCheckArchive">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="aUnpackArchive">
   <property name="text">
    <string>Unpack</string>
   </property>
  </action>
  <action name="aCreateArchive">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="aCheckingArchive">
   <property name="text">
    <string>Checking</string>
   </property>
  </action>
  <action name="aFileStitching">
   <property name="text">
    <string>Stitching</string>
   </property>
  </action>
  <action name="aSpectrumViewer">
   <property name="text">
    <string>Spectrum Viewer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
